File: app/src/main/java/com/example/nasaenterpriseapi/model/Collection.java
package com.example.nasaenterpriseapi.model;

import java.util.List;
import com.google.gson.annotations.SerializedName;

public class Collection{

	@SerializedName("metadata")
	private Metadata metadata;

	@SerializedName("href")
	private String href;

	@SerializedName("items")
	private List<Items> items;

	@SerializedName("version")
	private String version;

	public Metadata getMetadata(){
		return metadata;
	}

	public String getHref(){
		return href;
	}

	public List<Items> getItems(){
		return items;
	}

	public String getVersion(){
		return version;
	}
}

File: app/src/main/java/com/example/nasaenterpriseapi/model/NasaQueryResponse/Collection.kt
package com.example.nasaenterpriseapi.model.NasaQueryResponse

import com.google.gson.annotations.SerializedName

data class Collection (
    @SerializedName("version") val version : Double,
    @SerializedName("metadata") val metadata : Metadata,
    @SerializedName("href") val href : String,
    @SerializedName("items") val items : List<Items>
)
